projects/aca-shared/src/lib/components/document-base-page/document-base-page.component.ts

```diff
@@ -31,7 +31,7 @@ import { Store } from '@ngrx/store';
 import { MinimalNodeEntity, MinimalNodeEntryEntity, NodePaging } from '@alfresco/js-api';
```

Comment:
This file is moved from `aca-content/../page.components.ts` 

It is going to be used in both libraries, `aca-content` and `aca-preview` therefore I moved it to `shared` lib (so `aca-preview` does not depends on `aca-content`) 

ps. aca-preview is going to be creted [here](https://github.com/Alfresco/alfresco-content-app/pull/2859/files)